script/run-comp-match

```diff
@@ -12,5 +12,7 @@ sys.path.insert(1, str(REPO_ROOT / 'modules'))
 import match_utils  # isort:skip
 
 if __name__ == '__main__':
-    match = match_utils.Match()
-    match.main(match.parse_args())
+    try:
+        match_utils.Match()
+    except RuntimeError:
```

Comment:
I would discourage catching `RuntimeError`s. All sorts of things can raise this kind of error and generally if you don't know what's causing an error it's better to let it propagate to the caller.

I would also discourage catching errors at this place in the file, especially in this manner. Any error handling which is at the outer script level should probably instead either `exit` itself explicitly. Any other errors should either be converted to useful error messages and/or allowed to bubble so the user gets a traceback they can use to debug. This approach unfortunately masks everything about the error.